openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json

```diff
@@ -120,7 +120,7 @@
     "api-uppercase": "API",
     "api-uppercase-plural": "APIs",
     "app-analytic-plural": "App-Analytics",
-    "app-lowercase": "app",
+    "app-lowercase": "App",
```

Comment:
This should be "app" since it is lowercase.

Reply:
```suggestion
    "app-lowercase": "app",
```